rocAL/rocAL/include/pipeline/image.h

```diff
@@ -112,6 +112,9 @@ struct ImageInfo
     uint32_t * get_roi_height() const;
     const std::vector<uint32_t>& get_roi_width_vec() const;
     const std::vector<uint32_t>& get_roi_height_vec() const;
+    const std::vector<uint32_t>& get_orig_roi_width_vec() const;
```

Comment:
Would suggest changing the name to 
get_original_roi_width_vec()
get_original_roi_height_vec()

Reply:
Resolved in the latest commit